lib/cryptoexchange/exchanges/codex/market.rb

```diff
@@ -0,0 +1,8 @@
+module Cryptoexchange::Exchanges
+  module Codex
+    class Market < Cryptoexchange::Models::Market
+      NAME = 'codex'
+      API_URL = 'https://api.codex.one'
+    end
```

Comment:
`trade_page_url` must be implemented even if there isn't one. You can follow the same method structure but return the url to the exchange root page.

Reply:
done